package.json

```diff
@@ -88,6 +88,7 @@
     "@testing-library/user-event": "^12.1.3",
     "@types/angular": "1.6.56",
     "@types/angular-route": "1.7.0",
+    "@types/chance": "^1.1.1",
```

Comment:
Not sure we need the Chance lib but I can be convinced otherwise. If we were to keep this you need to use exact versions.

Reply:
I noticed it was also being used for the `jaeger-ui-components`: https://github.com/grafana/grafana/blob/master/packages/jaeger-ui-components/package.json#L29 so thought it might be nice to also use it here

I like its convenience for generating data for the mocks and thought it might be helpful for future tests but I don't mind removing it if we feel like it's unnecessary?

Reply:
Gotcha, maybe you'd like to propose this during next weeks Frontend weekly?

Reply:
Sounds good!